UnderlyingOptionsPS/FTPS3-Compare.ps1

```diff
@@ -118,9 +187,10 @@ foreach ($file in $filelist){
 
 	#If the item in the list contains a search term (from the file name) then keep it. 
 	#Some S3 buckets contain folders that would return bad matches. 
-	if ($file -match ($info.fileNamePrefix)) {     
-		$AWSlist.Add($AWSrx.Match($file.ToString()))
```

Comment:
This performs the matching twice, which isn't that big of a deal, but the change is more efficient.